src/modules/iotjs_module_gpio.h

```diff
@@ -63,7 +71,12 @@ typedef struct {
   uint32_t pin;
   GpioDirection direction;
   GpioMode mode;
-#if defined(__TIZENRT__)
+  GpioEdge edge;
+#if defined(__linux__)
+  int value_fd;
+  uv_thread_t thread;
+  uv_mutex_t mutex;
```

Comment:
Never mind, the struct is allocated with `IOTJS_ALLOC`, so it will be on the heap anyway.